src/main/com/fulcrologic/fulcro/algorithms/do_not_use/log_once.cljc

```diff
@@ -0,0 +1,18 @@
+(ns com.fulcrologic.fulcro.algorithms.do-not-use.log-once
+  "Some misc. logging functions. These are primarily meant for internal use, and are subject to
+  relocation and removal in the future.
+
+  You have been warned. Changes to this ns (or its complete removal)
+  will not be considered breaking changes to the library, and no mention of said changes
+  will even appear in the changelog."
+  (:require
+    [taoensso.encore :as encore]
+    [taoensso.timbre :as log]))
+
+; TODO Should we limit the size and use a LRU cache or st.? Hopefully not really an issue in a webapp?
+; Or implement a DIY TTL so we repeat the error sometimes? (=> {<args> (js/Date.now)})
+
+;(encore/memoize 60000 (constantly nil))
+
+(defn error-once! [& args]
```

Comment:
how about `log-once!` with level as first arg. The encore/memoize is sufficient.

Reply:
Thought I already made this comment somewhere, but it would be sufficient to have an arity 2 macro for log-once that elides with the same logic as timbre. Perhaps 2 macros: warn-once! and error-once!. The fixed arity then makes it possible for the macro to use a memoized (private) function.

Reply:
Also, let's convert timbre-support to a CLJC namespace, and put this stuff in there. These should be usable by libraries, and `do-not-use` really is meant to be something no one uses...though I know that isn't necessarily true :D